web/src/app/pages/home/home.controller.js

```diff
@@ -4,15 +4,20 @@
     angular.module('BlurAdmin.pages.home')
         .filter('split', function() {
             return function(input, splitChar, splitIndex) {
-                // do some bounds checking here to ensure it has that index
                 return input.split(splitChar)[splitIndex];
             };
         })
         .controller('HomeCtrl', HomeCtrl);
 
 
     /** @ngInject */
-    function HomeCtrl($http, $scope, printService, $state) {
+    function HomeCtrl($http, $scope, printService, $state,userService) {
```

Comment:
Code formatting issue where $state,userService needs a space.
Note: Its better to use the code formatter in IDE before committing the code